src/test/java/seedu/address/model/person/CovidStatusTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Test;
+
+public class CovidStatusTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new CovidStatus(null));
+    }
+
+    @Test
+    public void constructor_invalidCovidStatus_throwsIllegalArgumentException() {
+        String invalidCovidStatus = "";
+        assertThrows(IllegalArgumentException.class, () -> new CovidStatus(invalidCovidStatus));
+    }
+
+    @Test
+    public void isValidCovidStatus() {
+        //null faculty
```

Comment:
I think theres a typo here, should be covid status instead of faculty, applies to line 30 and 35

Reply:
Noted! I've made the appropriate changes 😄